pkg/web/static/js/main.js

```diff
@@ -30,15 +30,17 @@ document.body.addEventListener('htmx:responseError', (e) => {
   document.querySelector('.modal').close()
 
   // Display error response to user.
-  const error = JSON.parse(e.detail.xhr.response)
-  document.getElementById('toast')
-  .insertAdjacentHTML('beforeend', `
-  <div class="alert alert-error">
-    <i class="fa-solid fa-circle-xmark"></i>
-    <span>${error.error}</span>
-  </div>`)
+  if (e.detail.xhr.response !== '') {
```

Comment:
This change was made after receiving a JSON error after testing the reject implementation to ensure the parameters added `request_retry`.